java/java.hints/src/org/netbeans/modules/java/hints/jdk/mapreduce/TreeUtilities.java

```diff
@@ -48,4 +80,196 @@ public static boolean isPreOrPostfixOp(Tree.Kind opKind) {
                 || opKind == Tree.Kind.POSTFIX_DECREMENT
                 || opKind == Tree.Kind.PREFIX_DECREMENT;
     }
+
```

Comment:
Please put this into different utilities class, not inside mapreduce (as this is nothing really specific to mapreduce). error.Utilities would be a good please. Or a new Utilities class in jdk, up to you.

Reply:
Changes added